configure.ac

```diff
@@ -452,22 +434,7 @@ elif test "x$LIBSODIUM_FOUND" = "xno"; then
     AC_SUBST(LIBSODIUM_CFLAGS)
 fi
 
-# Checks for typedefs, structures, and compiler characteristics.
-AC_HEADER_STDBOOL
-AC_TYPE_INT16_T
-AC_TYPE_INT32_T
-AC_TYPE_PID_T
-AC_TYPE_SIZE_T
-AC_TYPE_UINT16_T
-AC_TYPE_UINT32_T
-AC_TYPE_UINT64_T
-AC_TYPE_UINT8_T
-AC_C_BIGENDIAN
-
-
 # Checks for library functions.
-AC_FUNC_FORK
-AC_CHECK_FUNCS([gettimeofday memset socket strchr malloc])
```

Comment:
It does seem these tests aren't used. But meanwhile it seems we should be testing (with AC_CHECK_FUNCS) for memset_s and explicit_bzero. CMakeLists checks for them, and crypto_core_mem.c uses the defines.

Reply:
Done.